msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -233,6 +250,12 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+1. You can remotely debug your app in Office app for Android. From your profile **(Me) > Settings > Enable remote debugging**, toggle on the option for **Enable remote debugging**.  In Office mobile Android, Enable remote debugging tool in Settings lets you see all sideloaded apps in the Apps menu.
```

Comment:
In Office mobile Android, Enable remote debugging tool in Settings also lets you see all sideloaded apps in the Apps menu.

Reply:
@harshatms did you mean you can see all a list of all running sideloaded apps as debuggable webviews from DevTools, or something else?

Reply:
In general, we show only mobile optimized apps, but if a developer wants to try out an unoptimized sideloaded app, they can enable remote debugging and then will be able to see it in Apps and finally can debug it as debugging requires app to be launched.